lib/string.js

```diff
@@ -376,6 +376,20 @@ internals.String = class extends Any {
         });
     }
 
+    base64() {
+
+        const regex = /^[a-z0-9\+\/]+(=+)?$/i;
```

Comment:
This regex doesn't seem to be good enough, there are few provided in http://stackoverflow.com/questions/475074/regex-to-parse-or-validate-base64-data, https://github.com/kevva/base64-regex seems quite good, don't think he'd be offended if we took it from him.